Started GET "/" for 127.0.0.1 at 2021-07-26 18:05:25 +0800
  
ActiveRecord::NoDatabaseError (Unknown database 'rails_shop_api_development'):
  
activerecord (6.1.4) lib/active_record/connection_adapters/mysql2_adapter.rb:43:in `rescue in new_client'
activerecord (6.1.4) lib/active_record/connection_adapters/mysql2_adapter.rb:39:in `new_client'
activerecord (6.1.4) lib/active_record/connection_adapters/mysql2_adapter.rb:23:in `mysql2_connection'
activerecord (6.1.4) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.4) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.4) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.4) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.4) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.4) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.4) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.4) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.4) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.4) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.4) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.4) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.4) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.4) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.4) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.4) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.4) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.4) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.4) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.4) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.4) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.4) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.4) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.4) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.4) lib/rails/engine.rb:539:in `call'
puma (5.3.2) lib/puma/configuration.rb:249:in `call'
puma (5.3.2) lib/puma/request.rb:77:in `block in handle_request'
puma (5.3.2) lib/puma/thread_pool.rb:338:in `with_force_shutdown'
puma (5.3.2) lib/puma/request.rb:76:in `handle_request'
puma (5.3.2) lib/puma/server.rb:438:in `process_client'
puma (5.3.2) lib/puma/thread_pool.rb:145:in `block in spawn_thread'
Started GET "/favicon.ico" for 127.0.0.1 at 2021-07-26 18:05:25 +0800
  
ActiveRecord::NoDatabaseError (Unknown database 'rails_shop_api_development'):
  
activerecord (6.1.4) lib/active_record/connection_adapters/mysql2_adapter.rb:43:in `rescue in new_client'
activerecord (6.1.4) lib/active_record/connection_adapters/mysql2_adapter.rb:39:in `new_client'
activerecord (6.1.4) lib/active_record/connection_adapters/mysql2_adapter.rb:23:in `mysql2_connection'
activerecord (6.1.4) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.4) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.4) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.4) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.4) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.4) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.4) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.4) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.4) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.4) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.4) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.4) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.4) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.4) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.4) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.4) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.4) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.4) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.4) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.4) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.4) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.4) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.4) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.4) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.4) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.4) lib/rails/engine.rb:539:in `call'
puma (5.3.2) lib/puma/configuration.rb:249:in `call'
puma (5.3.2) lib/puma/request.rb:77:in `block in handle_request'
puma (5.3.2) lib/puma/thread_pool.rb:338:in `with_force_shutdown'
puma (5.3.2) lib/puma/request.rb:76:in `handle_request'
puma (5.3.2) lib/puma/server.rb:438:in `process_client'
puma (5.3.2) lib/puma/thread_pool.rb:145:in `block in spawn_thread'
  [1m[35m (7.0ms)[0m  [1m[35mCREATE DATABASE `rails_shop_api_development` DEFAULT CHARACTER SET `utf8mb4`[0m
  [1m[35m (1.0ms)[0m  [1m[35mCREATE DATABASE `rails_shop_api_test` DEFAULT CHARACTER SET `utf8mb4`[0m
Started GET "/" for 127.0.0.1 at 2021-07-26 18:08:27 +0800
Processing by Rails::WelcomeController#index as HTML
  Rendering /Users/jolly/.rvm/gems/ruby-2.7.2/gems/railties-6.1.4/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /Users/jolly/.rvm/gems/ruby-2.7.2/gems/railties-6.1.4/lib/rails/templates/rails/welcome/index.html.erb (Duration: 3.5ms | Allocations: 294)
Completed 200 OK in 7ms (Views: 4.2ms | ActiveRecord: 0.0ms | Allocations: 2668)


Started GET "/favicon.ico" for 127.0.0.1 at 2021-07-26 18:08:27 +0800
  
ActionController::RoutingError (No route matches [GET] "/favicon.ico"):
  
Started GET "/" for 127.0.0.1 at 2021-07-26 18:08:47 +0800
Processing by Rails::WelcomeController#index as HTML
  Rendering /Users/jolly/.rvm/gems/ruby-2.7.2/gems/railties-6.1.4/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /Users/jolly/.rvm/gems/ruby-2.7.2/gems/railties-6.1.4/lib/rails/templates/rails/welcome/index.html.erb (Duration: 7.1ms | Allocations: 178)
Completed 200 OK in 10ms (Views: 8.2ms | ActiveRecord: 0.0ms | Allocations: 787)


Started GET "/favicon.ico" for 127.0.0.1 at 2021-07-26 18:08:48 +0800
  
ActionController::RoutingError (No route matches [GET] "/favicon.ico"):
  
